Add release notes for 1.15.3 (#176)
anbox-cloud/discourse-docs: announcements/1.15.3.md

### Introduction

The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.15.3.

> Please note that this is an out of order release which only applies to the Anbox Cloud Appliance.

Reviewer: ```suggestion
Note that this is an out of order release that only applies to the Anbox Cloud Appliance.
```

Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.

#### New features & improvements

> * Workaround a bug in Juju 2.9.35 which prevents the Anbox Cloud Appliance deploying its software stack entirely. The workaround includes a Juju binary of a previous release inside the appliance snap to make the deployment process independent of the Juju snap installed on the host system.

Reviewer: ```suggestion
* Work around a bug in Juju 2.9.35 that prevents the Anbox Cloud Appliance from deploying its software stack entirely. The workaround makes the deployment process independent of the Juju snap that is installed on the host system by including a Juju binary of a previous release inside the appliance snap.
```
